tests/functional/cylc-clean/03-multi.t

```diff
@@ -75,13 +75,13 @@ ${HOME}/cylc-run/${CYLC_TEST_REG_BASE}
 __TREE__
 
 mkdir "${WORKFLOW_RUN_DIR}/run1/.service"
-touch "${WORKFLOW_RUN_DIR}/run1/.service/db"  # corrupted db!
+echo 'x' > "${WORKFLOW_RUN_DIR}/run1/.service/db"  # corrupted db!
```

Comment:
I don't think that is a valid way to imitate a corrupted DB. Instead it makes it a non-DB file, whereas the way it was before (a blank file) is accepted as a DB by sqlite3

Reply:
I'm not sure a blank file is a legit way to make a blank DB as SQLITE will happily take a blank file and turn it into a DB.

Reply:
Ok. I guess I was thinking more in terms of the DB contents being "corrupted" (e.g. empty tables which has happened before in #4637) rather than the file itself becoming unreadable